src/containers/LoginForm.test.js

```diff
@@ -1,13 +1,219 @@
+        wrapper.find('#email-input').simulate('change', mockEvent);
+        expect(wrapper.instance().handleInputChange)
+          .toHaveBeenCalledWith(mockEvent);
+
+        wrapper.find('#password-input').simulate('change', mockEvent);
+        expect(wrapper.instance().handleInputChange)
+          .toHaveBeenCalledWith(mockEvent);
+    });
+
+    it('should invoke validateInputs when handleSubmit is called', () => {
+      const mockEvent = { preventDefault: jest.fn() };
+      wrapper.instance().validateInputs = jest.fn();
+      wrapper.instance().handleSubmit(mockEvent);
+      expect(wrapper.instance().validateInputs).toHaveBeenCalled();
+    });
+
+    it('should setState when validateInputs is called with an input empty',
+      () => {
+        const mockStartingState = {email: 'fakeemail@email.com',
+          password: '',
+          error: ''};
+        const mockEndingState = {email: 'fakeemail@email.com',
+          password: '',
+          error: 'All input fields required'};
+        wrapper.setState(mockStartingState);
+        expect(wrapper.state()).toEqual(mockStartingState);
+
+        wrapper.instance().validateInputs();
+        expect(wrapper.state()).toEqual(mockEndingState);
+    });
+
+    it('should setState when handleInputChange is called', () => {
+      const mockEvent = { target: { name: 'email', value: 'testValue' } };
+      const mockStartingState = {
+        email: 'fakeemail@email.com',
+        password: 'fakepassword',
+        error: 'fakeerror' };
+      const mockEndingState = {
+        email: 'testValue',
+        password: 'fakepassword',
+        error: '' };
+      wrapper.setState(mockStartingState);
+      wrapper.instance().handleInputChange(mockEvent);
+      expect(wrapper.state()).toEqual(mockEndingState);
+    });
+
+    it('should invoke handleError prop method when handleInputChange is called',
+      () => {
+        const mockEvent = { target: { name: 'email', value: 'testValue' } };
+        wrapper.instance().handleInputChange(mockEvent);
+        expect(mockHandleError).toHaveBeenCalled();
+    });
+
+    it('should invoke fetchUserLogin when handleSubmit is called', () => {
+      const mockStartingState = {
+        email: 'Marge@turing.io',
+        password: 'fakepassword',
+        error: '' };
+      wrapper.setState(mockStartingState);
+
+      const mockEvent = { preventDefault: jest.fn() };
+      wrapper.instance().handleSubmit(mockEvent);
+      expect(fetchUserLogin)
+        .toHaveBeenCalledWith('Marge@turing.io', 'fakepassword');
+    });
+
+    it('should invoke fetchRatings when fetchUserLogin resolves', async () => {
+      await fetchUserLogin('Marge@turing.io', 'fakepassword');
+      expect(fetchRatings).toHaveBeenCalledWith(9);
+    });
+
+    // it('should invoke updateLoggedInStatus with true when fetchRatings has resolved', () => {
+    // });
+    //
+    // it('should invoke updateUser with data passed in when fetchRatings has resolved', () => {
```

Comment:
I am having trouble getting these two tests to pass.  It shows that the number of calls for updateLoggedInStatus / updateUser respectively is 0.  I've tried invoking handleSubmit, fetchUserLogin, fetchRatings and tried moving async/await around without success.  Any ideas?